package com.res_gatinhos.api.application.exceptions

class VoluntarioNaoEncontradoException(
    message: String = "Voluntario n√£o encontrado."
): RuntimeException(
    message = message
)